openreview/conference/matching.py

```diff
@@ -250,7 +250,13 @@ def _build_subject_area_scores(self, submissions):
         edges = []
         user_subject_areas = list(openreview.tools.iterget_notes(
             self.client,
-            invitation=self.conference.get_registration_id()))
+            invitation=self.conference.get_registration_id(self.conference.get_reviewers_id())))
```

Comment:
this has to be done once and it depends on `match_group` property of the matching class:

```
invitation=self.conference.get_registration_id(self.match_group.id)
```